app/models/hubstats/deploy.rb

```diff
@@ -0,0 +1,36 @@
+module Hubstats
+	class Deploy < ActiveRecord::Base
```

Comment:
Something is funky with your formatting in this file.  Try to re-format it so the tab indentation is corrected.